release/_package.sh

```diff
@@ -19,6 +19,55 @@ function generate_checksum_files {
 	done
 }
 
+function generate_javadocs {
+	if [[ "${_PRODUCT_VERSION}" != 7.3.*-ga* ]] &&
+	   [[ "${_PRODUCT_VERSION}" != 7.3.*-u* ]] &&
+	   [[ "${_PRODUCT_VERSION}" != 7.4.*-ga* ]]
+	then
+		lc_log INFO "Javadocs should not be generated for ${_PRODUCT_VERSION}."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+
+	lc_log INFO "Generating javadocs for ${_PRODUCT_VERSION}."
+
+	git reset --hard && git clean -dfx
+
+	git fetch upstream "refs/tags/${_PRODUCT_VERSION}:refs/tags/${_PRODUCT_VERSION}"
+
+	git checkout "tags/${_PRODUCT_VERSION}"
+
+	local portal_release_edition_private="true"
+
+	if [ "${LIFERAY_RELEASE_PRODUCT_NAME}" == "portal" ]
+	then
+		portal_release_edition_private="false"
+	fi
+
+	local release_info_version="$(echo "${_PRODUCT_VERSION}" | cut -d '-' -f 1)"
+
+	local release_info_version_file_suffix="-$(echo "${_PRODUCT_VERSION}" | cut -d '-' -f 2)"
+
+	ant \
+		-Ddist.dir="${_BUILD_DIR}/release" \
+		-Dliferay.product.name="liferay-${LIFERAY_RELEASE_PRODUCT_NAME}" \
+		-Dlp.version="${_PRODUCT_VERSION}" \
+		-Dpatch.doc="true" \
+		-Dportal.dir="${_PROJECTS_DIR}/liferay-portal-ee" \
+		-Dportal.release.edition.private="${portal_release_edition_private}" \
+		-Drelease.info.version="${release_info_version}" \
+		-Dservice.pack.version.suffix="${release_info_version_file_suffix}" \
```

Comment:
@kiwm, please name the variable `release_info_version_file_suffix` as `service_pack_version_suffix`